### PR TITLE
Examples from support

### PR DESCRIPTION
**Pull request by tristen**
A bunch of examples based on answering some support questions last week I've turned into proper examples.

### CODE REVIEW

**Comment by jfirebaugh** [comment]
What the use case for disable-UTFGrid-interaction example? Would it be better to just use:

```
var map = L.mapbox.map('map', 'examples.map-8ced9urs', {gridControl: false});
```

**Comment by jfirebaugh** [comment]
[Toggling UI](https://www.mapbox.com/mapbox.js/example/v1.0.0/toggling-ui/) also shows how to toggle a legend, albeit it has a lot of other features as well.

Is geojson-marker-from-remote-url substantially different than [GeoJSON Marker from URL](https://www.mapbox.com/mapbox.js/example/v1.0.0/geojson-marker-from-url/)?

**Comment by tmcw** [comment]
- Hm, loading from raw.github.com in an example seems like a very bad idea.
- marker-navigation-from-menu-list.html should not use data attributes, it should use variables.
- marker-navigation-from-menu-list.html , on second glance, will not run without errors.

**Comment by tristen** [comment]
Thanks for the review @jfirebaugh 

> What the use case for disable-UTFGrid-interaction example?

The use case would be disabling the default interaction in order to roll with your own thing?

> Toggling UI also shows how to toggle a legend, albeit it has a lot of other features as well.

Totally fine with dropping that one.

> Is geojson-marker-from-remote-url substantially different than GeoJSON Marker from URL?

geojson-marker-from-remote-url demonstrates pulling a geojson file from github which is different from a local resource.

File: _posts/examples/v1.0.0/0100-01-01-geojson-marker-from-remote-url.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+<script>
+var url = 'https://api.github.com/repos/mapbox/mapbox.js/contents/test/manual/example.geojson';
+
+var map = L.mapbox.map('map', 'examples.map-h67hf2ic')
+  .setView([0.3941, -78.2227], 7);
+
+function load() {
+  // Fetch just the contents of a .geojson file from GitHub by passing
+  // `application/vnd.github.v3.raw` to the Accept header
```
Comment (tmcw) [inline _posts/examples/v1.0.0/0100-01-01-geojson-marker-from-remote-url.html:22]: Add a note about this using CORS and not guaranteed to work in all browsers. It's more important to explain the dumb parts than the complicated parts of examples.

File: _posts/examples/v1.0.0/0100-01-01-geojson-marker-from-remote-url.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  $.ajax({
+    headers: {
+      'Accept': 'application/vnd.github.v3.raw'
+    },
+    dataType: 'json',
+    url: url,
+    success: function(geojson) {
+        // On success add fetched data to the map.
+        var remoteLayer = L.mapbox.featureLayer().addTo(map);
```
Comment (tmcw) [inline _posts/examples/v1.0.0/0100-01-01-geojson-marker-from-remote-url.html]: Consolidate these two calls: the first argument to `featureLayer` can be a geojson object.

File: _posts/examples/v1.0.0/0100-01-01-marker-popup-onload.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+description: Control whether a marker tooltip should be displayed on load
+tags: [ui, markers]
+---
+
+<div id='map'></div>
+<script>
+var map = L.mapbox.map('map', 'examples.map-h67hf2ic');
+map.setView([40.7271, -73.9840], 14);
+
+// Build a marker from a simple .geojson object:
```
Comment (tmcw) [inline _posts/examples/v1.0.0/0100-01-01-marker-popup-onload.html]: should be GeoJSON, not .geojson

**Pull request merged**